app/vue/contexts/competition/SectionLeaderboardContext.js

```diff
@@ -124,6 +124,40 @@ export default class SectionLeaderboardContext extends BaseFuroContext {
     URL.revokeObjectURL(blobURL)
   }
 
+  /**
+   * Normalize equity.
+   *
+   * @param {{
+   *   equity: number
+   *   statusId: number
+   * }} params - Parameters.
+   * @returns {string | number}
+   */
+  normalizeEquity ({
+    equity,
+    statusId,
+  }) {
+    if (statusId === COMPETITION_PARTICIPANT_STATUS.AWAITING_DEPOSIT.ID) {
```

Comment:
```suggestion
    if (statusId !== COMPETITION_PARTICIPANT_STATUS.AWAITING_DEPOSIT.ID) {
```

We need to show the equity if the status is `AWAITING_DEPOSIT`